services/core-web/src/components/Forms/permits/conditions/AddCondition.js

```diff
@@ -68,12 +69,15 @@ export class AddCondition extends Component {
   };
 
   render = () => {
+    const title = this.props.initialValues.alternateTitle
```

Comment:
Shouldn't this be `this.props.alternateTitle`? Looking at the `defaultProps` above.

Reply:
yeah I'll fix this

Also, speaking of alternate titles do you have any suggestions for the 6th layer? 
Sections > Sub Sections > Condtions > List Items > Sub-List Items > ???

Reply:
this title has been removed.